internal/uidriver/glfw/ui.go

```diff
@@ -670,11 +670,10 @@ func (u *UserInterface) run(context driver.UIContext) error {
 }
 
 func (u *UserInterface) updateSize(context driver.UIContext) {
-	var w, h int
-	_ = u.t.Call(func() error {
-		w, h = u.windowWidth, u.windowHeight
-		return nil
+	r1, r2 := u.t.CallReturn2(func() (uintptr, uintptr) {
```

Comment:
If this call matters in terms of heap allocation, what about using sync.Mutex or atomic.Value?